### PR TITLE
Integrate identity pallet

### PR DESCRIPTION
**Pull request by ivanceras**
Resolved issue #189 

Request for comment on:
- type Slashed = ()
- identity pallet is added to the runtime, is this a breaking change?

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -80,7 +81,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
     spec_version: 226,
-    impl_version: 1,
+    impl_version: 2,
```
Comment (philipstanislaus) [inline runtime/src/lib.rs:84]: I’d say this constitutes a spec version change?

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -516,6 +517,27 @@ impl substrate_pallet_multi_account::Trait for Runtime {
     type MultisigDepositFactor = MultiAccountSigDepositFactor;
 }
 
+parameter_types! {
+    pub const MaxSubAccounts: u32 = 100;
+    pub const MaxAdditionalFields: u32 = 100;
+    pub const BasicDeposit: Balance = 10 * MICRO_RAD;
+    pub const FieldDeposit: Balance = 250 * MICRO_RAD / 100;
+    pub const SubAccountDeposit: Balance = 2 * MICRO_RAD;
```
Comment (philipstanislaus) [inline runtime/src/lib.rs:525]: See discussion on Slack

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -516,6 +517,27 @@ impl substrate_pallet_multi_account::Trait for Runtime {
...
+
+impl pallet_identity::Trait for Runtime {
+    type Event = Event;
+    type Currency = Balances;
+    type BasicDeposit = BasicDeposit;
+    type FieldDeposit = FieldDeposit;
+    type SubAccountDeposit = SubAccountDeposit;
+    type MaxSubAccounts = MaxSubAccounts;
+    type MaxAdditionalFields = MaxAdditionalFields;
+    type Slashed = ();
```
Comment (philipstanislaus) [inline runtime/src/lib.rs:536]: That is fine, it means we burn on slash.

File: runtime/src/lib.rs (hunk 1 above)
Comment (ivanceras) [reply runtime/src/lib.rs:84]: I think so, since the `construct_runtime!` is `pub` and has a new field: `Identity`

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -544,6 +566,7 @@ construct_runtime!(
 		Fees: fees::{Module, Call, Storage, Event<T>, Config<T>},
 		Nfts: nfts::{Module, Call, Event<T>},
 		MultiAccount: substrate_pallet_multi_account::{Module, Call, Storage, Event<T>, Config<T>},
+                Identity: pallet_identity::{Module, Call, Storage, Event<T>},
```
Comment (vedhavyas) [inline runtime/src/lib.rs:569]: indentation ?

**Comment by ivanceras** [comment]
@philipstanislaus do we merge this to master or to a separate branch(`edge`)?

**Pull request merged**